tests/SprykerEcoTest/Zed/Unzer/_support/UnzerZedTester.php

```diff
@@ -0,0 +1,565 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEcoTest\Zed\Unzer;
+
+use Codeception\Actor;
+use Codeception\Scenario;
+use Generated\Shared\DataBuilder\QuoteBuilder;
+use Generated\Shared\DataBuilder\UnzerCredentialsBuilder;
+use Generated\Shared\DataBuilder\UnzerPaymentBuilder;
+use Generated\Shared\Transfer\PaymentTransfer;
+use Generated\Shared\Transfer\QuoteTransfer;
+use Generated\Shared\Transfer\SaveOrderTransfer;
+use Generated\Shared\Transfer\StoreRelationTransfer;
+use Generated\Shared\Transfer\StoreTransfer;
+use Generated\Shared\Transfer\UnzerApiCreateBasketResponseTransfer;
+use Generated\Shared\Transfer\UnzerApiCreateCustomerResponseTransfer;
+use Generated\Shared\Transfer\UnzerApiCreateMetadataResponseTransfer;
+use Generated\Shared\Transfer\UnzerApiCreatePaymentResourceResponseTransfer;
+use Generated\Shared\Transfer\UnzerApiGetPaymentResponseTransfer;
+use Generated\Shared\Transfer\UnzerApiResponseTransfer;
+use Generated\Shared\Transfer\UnzerApiSetWebhookResponseTransfer;
+use Generated\Shared\Transfer\UnzerApiUpdateCustomerResponseTransfer;
+use Generated\Shared\Transfer\UnzerBasketTransfer;
+use Generated\Shared\Transfer\UnzerCredentialsResponseTransfer;
+use Generated\Shared\Transfer\UnzerCredentialsTransfer;
+use Generated\Shared\Transfer\UnzerCustomerTransfer;
+use Generated\Shared\Transfer\UnzerKeypairTransfer;
+use Generated\Shared\Transfer\UnzerNotificationTransfer;
+use Generated\Shared\Transfer\UnzerPaymentTransfer;
+use Spryker\Shared\Vault\VaultConstants;
+use SprykerEco\Shared\Unzer\UnzerConstants;
+use SprykerEco\Zed\Unzer\Business\UnzerFacadeInterface;
+use SprykerEco\Zed\Unzer\Persistence\UnzerEntityManager;
+use SprykerEco\Zed\Unzer\Persistence\UnzerRepository;
+use SprykerEco\Zed\Unzer\UnzerConfig;
+
+/**
+ * Inherited Methods
+ *
+ * @method void wantToTest($text)
+ * @method void wantTo($text)
+ * @method void execute($callable)
+ * @method void expectTo($prediction)
+ * @method void expect($prediction)
+ * @method void amGoingTo($argumentation)
+ * @method void am($role)
+ * @method void lookForwardTo($achieveValue)
+ * @method void comment($description)
+ * @method \Codeception\Lib\Friend haveFriend($name, $actorClass = NULL)
+ *
+ * @SuppressWarnings(PHPMD)
+ */
+class UnzerZedTester extends Actor
+{
+    use _generated\UnzerZedTesterActions;
+
+    /**
+     * @var string
+     */
+    public const PAYMENT_SELECTION_MARKETPLACE = 'unzerMarketplaceBankTransfer';
+
+    /**
+     * @var int
+     */
+    public const TOTALS_PRICE_TO_PAY = 72350;
+
+    /**
+     * @var string
+     */
+    public const ADDRESS_COUNTRY = 'Germany';
```

Comment:
I don't see usage outside of this class, can we make it `protected`?
Please check similar cases below.